snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: wifi-ap-client
+version: 1
+confinement: strict
+summary: HTTP over Unix
```

Comment:
Lets put a proper summary of what snap does here. Same goes for the description field.